features/messages/requests/MarkAllReadButton.tsx

```diff
@@ -73,15 +72,10 @@ export default function MarkAllReadButton({
       {markAll.error && (
         <Snackbar severity="error">{markAll.error.message}</Snackbar>
       )}
-      <Tooltip title={t("mark_all_read_button_text")}>
-        <IconButton
-          aria-label={t("mark_all_read_button_text")}
-          loading={markAll.isLoading}
-          onClick={() => markAll.mutate()}
-        >
-          <DoneAllIcon />
-        </IconButton>
-      </Tooltip>
+
+      <Button loading={markAll.isLoading} variant="text" onClick={() => markAll.mutate()}>
+        <DoneAllIcon /> {t("mark_all_read_button_text")}
```

Comment:
Probably a small nitpick, but I think it might be worth wrapping the text around a Typography and add a tiny bit of padding between the icon and the text

Reply:
I agree with @darrenvong ! 

<img width="343" alt="Screen Shot 2022-09-19 at 12 17 20 PM" src="https://user-images.githubusercontent.com/83376913/191075539-f2b39c29-166c-44e5-b6fa-b6bc1efb38cd.png">